tests/conftest.py

```diff
@@ -271,14 +271,14 @@ def validate_counts(shots, results1, results2, rtol=20, atol=0.2):
             assert np.allclose(val1, val2, rtol=rtol, atol=atol)
 
 
-def validate_samples(shots, results1, results2, rtol=20, atol=0.2):
+def validate_samples(shots, results1, results2, rtol=0.15, atol=30):
     """Compares two samples.
 
     If the results are ``Sequence``s, loop over entries.
 
-    Fails if the results do not have the same shape, within 15 percent plus ``20`` entries.
+    Fails if the results do not have the same shape, within 15 percent plus ``30`` entries.
     This is to handle cases when post-selection yields variable shapes.
-    Otherwise, fails if the sums of samples differ by more than 15 percent plus ``20``.
+    Otherwise, fails if the sums of samples differ by more than 15 percent plus ``30``.
```

Comment:
Is there a principled reason for the 15% + 30 number, or does this just seem good in testing?